build-tools/build.gradle.kts

```diff
@@ -61,7 +61,7 @@ dependencies {
     api("org.jetbrains.kotlin:kotlin-native-shared:$konanVersion")
     implementation("com.github.jengelman.gradle.plugins:shadow:5.1.0")
 
-    implementation("org.jetbrains.kotlinx:kotlinx-metadata-klib:0.0.1-dev-9")
+    implementation("org.jetbrains.kotlinx:kotlinx-metadata-klib:0.0.1-dev-10")
```

Comment:
It seems that kotlinx-metadata's version should be in `gradle.properties` file as its version is necessary for the compiler.